notification-template/blade/DataSource/bin/start-jar.sh

```diff
@@ -28,6 +28,7 @@ if [ $confreading = 1 ]; then
    java -jar "$jar" "$@"
    exit $?
 else
-   java -cp "$classpath" -jar "$jar" "$@" 2> "$LOGDIR"/java-$BLADENAME.log >/dev/null &
+   start_command="java -cp "$classpath" -jar "$GC_OPTIONS" "$jar" "$@" 2> "$LOGDIR"/java-$BLADENAME.log >/dev/null &"
```

Comment:
GC_OPTIONS is too general. CAPLIN_BLADE_JAVA_OPTIONS is better.
The customer could set an environment variable named GC_OPTIONS and then we would clash.
Having CAPLIN_ at the start makes it less likely and makes it clear the environment variable is for Caplin components.
BLADE indicates that it's directed at blades
JAVA_OPTIONS indicates that any Java options could be set - GC and/or others